classes/StandardControlsRegistry.php

```diff
@@ -734,6 +734,22 @@ protected function registerDatepickerWidget()
                 'type' => 'string',
                 'ignoreIfEmpty' => true,
                 'sortOrder' => 85
+            ],
+            'firstDay' => [
+                'title' => Lang::get('winter.builder::lang.form.property_datepicker_first_day'),
+                'type' => 'dropdown',
+                'default' => 'datetime',
```

Comment:
@arvislacis This needs to default to Sunday (`"0"`).

```suggestion
                'default' => '0',
```